docs/book/getting_started/installation_and_setup.md

```diff
@@ -25,6 +25,8 @@ mv kubebuilder_${version}_darwin_${arch} kubebuilder && sudo mv kubebuilder /usr
 export PATH=$PATH:/usr/local/kubebuilder/bin
 ```
 
+Don't use homebrew to install kubebuilder. The [homebrew recipe is broken](https://github.com/Homebrew/homebrew-core/issues/44175).
```

Comment:
WDYT about we fix the formula instead of adding this note?

Reply:
I agree, in any homebrew issue we should fix the formula rather than the documentation. I still think the homebrew formula works as expected here, though?